main.tf

```diff
@@ -86,7 +86,7 @@ module "rosa_cluster_classic" {
   source = "./modules/rosa-cluster-classic"
 
   cluster_name               = var.cluster_name
-  operator_role_prefix       = local.operator_role_prefix
+  operator_role_prefix       = var.create_operator_roles ? module.operator_roles[0].operator_role_prefix : local.operator_role_prefix
```

Comment:
Shouldn't that be the other way around? If you are creating the operator roles take it from local else take from the existing values of the module?

Reply:
The local is just a way to get the value provided from user with default value, in case it wasn't provided.
The only reason using the output from the operator role is to have dependency between the modules, so the cluster creation can't start before the operator roles creation completed.
In case the operator roles are pre-created by the user, its not needed